buildSrc/src/tasks/README.md

```diff
@@ -0,0 +1,5 @@
+# build configuration
+All gradle files in this folder (but not in subdirectories) will be added to the gradle build configuration. They will be added in the order indicated by their file name, that’s why numbers are used to order them. The files should be kept at a reasonable small size. If more complicated logic is required, it should be put in groovy classes.
+
+## task dependencies
```

Comment:
Can you use capital letters for the titles, like in all other docs, too, please?